packages/protocol/src/action-protocol/clipboard.ts

```diff
@@ -14,80 +14,154 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { isObject, isString } from '../utils/typeguard-util';
-import { generateRequestId, isActionKind, Operation, RequestAction, ResponseAction } from './base-protocol';
+import { hasObjectProp } from '../utils/type-util';
+import { Action, Operation, RequestAction, ResponseAction } from './base-protocol';
 import { EditorContext } from './types';
 
 /**
  * Requests the clipboard data for the current editor context, i.e., the selected elements, in a clipboard-compatible format.
+ * The corresponding namespace declares the action kind as constant and offers helper functions for type guard checks
+ * and creating new `RequestClipboardDataActions`.
  */
-export class RequestClipboardDataAction implements RequestAction<SetClipboardDataAction> {
-    static readonly KIND = 'requestClipboardData';
+export interface RequestClipboardDataAction extends RequestAction<SetClipboardDataAction> {
+    /**
+     * The unique action kind.
+     */
```

Comment:
I'd spare such comments, also if it hurts to be "incomplete". But imho they just add noise and add no information.